### ISSUE TITLE
Point navigation breadcrumbs

### ISSUE THREAD

**Issue by connormcmk** [network-goods-institute/negation-game#28]
Clicking a point while already having one open should add it to the lineage, a stack that sits above the point

This way a user can see the string of arguments that got them to this point and return back up to them
<img width="637" alt="Screenshot 2024-11-04 at 6 08 08 PM" src="https://github.com/user-attachments/assets/012ee848-095c-4900-b152-730a956a2f16">

When a point was already opened as an ancestor to opening a particular point, it should not show up in the list of negations (or at least it should be made visually obvious that it has been visited in this lineage already)

**Comment by volkyeth** [comment]
Makes sense, but not sure if there's a mobile-friendly solution. Most Points will only make sense if we display the full content, and stacking those up as breadcrumbs will take up a lot of real estate.

On larger screens it's easier, since we could use another column for context, or we could highlight the trail in a graph view

 I'll try my hand at it.
